From c7f3e2a8fe530beec6103cb9071ccc41458879aa Mon Sep 17 00:00:00 2001
From: Tony Tascioglu <tony.tascioglu@windriver.com>
Date: Mon, 21 Jun 2021 12:34:22 -0400
Subject: [PATCH] libyang: fix initial build errors

This patch addresses build errors seen when integrating libyang as a
recipe.

There is a missing include statement for stdint.h in utests.h which
causes build problems. stdint.h is required to be imported by cmocka.h
prior to the cmocka file. Adding the stdint allows it to build the
tests correctly

Upstream-Status: Submitted [ https://github.com/CESNET/libyang/pull/1819 ]

Signed-off-by: Tony Tascioglu <tony.tascioglu@windriver.com>
Signed-off-by: Randy MacLeod <randy.macleod@windriver.com>
---
 tests/utests/utests.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/utests/utests.h b/tests/utests/utests.h
index 7e85a66e..8eee26ce 100644
--- a/tests/utests/utests.h
+++ b/tests/utests/utests.h
@@ -22,6 +22,7 @@
 #include <stdarg.h>
 #include <stddef.h>
 #include <stdlib.h>
+#include <stdint.h>
 
 #include <cmocka.h>
 
-- 
2.32.0
